lib/DDG/Spice/MarsPhotos.pm

```diff
@@ -0,0 +1,36 @@
+package DDG::Spice::MarsPhotos;
+use DDG::Spice;
+
+spice is_cached => 1;
+spice proxy_cache_valid => "200 1d";
+spice wrap_jsonp_callback => 1;
+
+# get real key later
+spice to => 'https://api.nasa.gov/mars-photos/api/v1/rovers/$1/photos?earth_date=$2&api_key=DEMO_KEY';
+spice from => '([^/]+)/?(?:([^/]+)/?(?:([^/]+)|)|)';
+
+triggers any => 'mars images', 'mars photos';
+
+handle remainder => sub {
+    my $remainder = $_;
```

Comment:
we may need to use a regex to validate the query matches some specific patterns. Otherwise this is prone to over triggering...